indexd/default_settings.py

```diff
@@ -11,7 +11,8 @@
 SQLALCHEMY_VERBOSE = (
     os.getenv('INDEXD_VERBOSE', '').lower() in ['1', 'yes', 'true']
 )
-PG_URL = 'postgresql://test:test@localhost/indexd_test'
+PG_HOST = os.getenv("PG_INDEXD_HOST", "localhost")
```

Comment:
All the way in here and didn't want to extract the user (test) and password (test) to environment variables?

Reply:
LOL, sorry  I was in a hurry to solve my problem and did not notice that.